### PR TITLE
Allow user scripts execution at scheduler start

### PR DESCRIPTION
**Pull request by mbenguig**

### CODE REVIEW

File: common/common-api/src/main/java/org/ow2/proactive/core/properties/PACommonPropertiesHelper.java [hunk 1]
```diff
@@ -105,6 +106,13 @@ public synchronized void updateProperties(String filename) {
     }
 
     /**
+     * Return all properties as a HashMap.
+     */
+    public HashMap<String,Object> getPropertiesAsHashMap() {
```
Comment (fviale) [inline common/common-api/src/main/java/org/ow2/proactive/core/properties/PACommonPropertiesHelper.java]: getPropertiesAsMap, return Map interface

**Comment by sonarqube-activeeon** [comment]
SonarQube analysis reported 1 issue

1. MINOR [SchedulerStarter.java#L221](https://github.com/ow2-proactive/scheduling/blob/58729d43e6ba4db35a09b486acb57815b6773d74/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/util/SchedulerStarter.java#L221): Define and throw a dedicated exception instead of using a generic one. [rule](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS00112)

**Comment by sonarqube-activeeon** [comment]
SonarQube analysis reported 1 issue

1. MINOR [SchedulerStarter.java#L221](https://github.com/ow2-proactive/scheduling/blob/8f1c7518f1a0b08e640eba3429eca336fc31a6d0/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/util/SchedulerStarter.java#L221): Define and throw a dedicated exception instead of using a generic one. [rule](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS00112)

**Comment by sonarqube-activeeon** [comment]
SonarQube analysis reported 5 issues

1. MINOR [LiveJobs.java#L672](https://github.com/ow2-proactive/scheduling/blob/fd46f0cf1e6ff277126734beecfe10e784172ea6/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/LiveJobs.java#L672): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. MINOR [LiveJobs.java#L760](https://github.com/ow2-proactive/scheduling/blob/fd46f0cf1e6ff277126734beecfe10e784172ea6/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/LiveJobs.java#L760): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. MINOR [LiveJobs.java#L815](https://github.com/ow2-proactive/scheduling/blob/fd46f0cf1e6ff277126734beecfe10e784172ea6/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/LiveJobs.java#L815): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. MINOR [LiveJobs.java#L851](https://github.com/ow2-proactive/scheduling/blob/fd46f0cf1e6ff277126734beecfe10e784172ea6/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/LiveJobs.java#L851): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. MINOR [SchedulerStarter.java#L223](https://github.com/ow2-proactive/scheduling/blob/fd46f0cf1e6ff277126734beecfe10e784172ea6/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/util/SchedulerStarter.java#L223): Define and throw a dedicated exception instead of using a generic one. [rule](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS00112)

**Comment by sonarqube-activeeon** [comment]
SonarQube analysis reported 1 issue

1. MINOR [SchedulerStarter.java#L223](https://github.com/ow2-proactive/scheduling/blob/360e63c3f40fc6041933e4c3255b2a88848c1284/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/util/SchedulerStarter.java#L223): Define and throw a dedicated exception instead of using a generic one. [rule](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS00112)

**Pull request merged**